build_product_images.py

```diff
@@ -45,7 +45,7 @@ def parse_args():
     )
     parser.add_argument("-u", "--push", help="Push images", action="store_true")
     parser.add_argument("-d", "--dry", help="Dry run.", action="store_true")
-    parser.add_argument("-a", "--architecture", help="Target platform for image, Expecting -a <platform 1> <platform 2> ...", nargs='+')
+    parser.add_argument("-a", "--architecture", help="Target platform for image, Expecting -a <platform 1> <platform 2> ... At least one argument", nargs='+', required=True)
```

Comment:
I would add a little validation as here as described here: https://stackoverflow.com/a/37472037

We also have an example in the (now defunct) integration-tests repo: https://github.com/stackabletech/integration-tests/blob/8114543fc51b7cd66cec50435cb791b274babec7/create_test_cluster.py#L116